index.html

```diff
@@ -26,7 +26,7 @@
             <div class="icons">
```

Comment:
The second section is the headline section of our page, so please use a `<section>` tag to contain this whole section, since It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.
Also please use semantic elemts to display proper contain a title can be enclosed in a `h1` tag, a text can use a` p `tag and so on, This way will be semantically correct and the screen readers and other technologies will thank you.
```
<section cladss="headline>
<h1> main title <h1>
<h2> sub title </h3>
<p> text <p>
<\section>
```